### DISCUSSION TITLE
Any way to filter on tags for Cognito identity-pool or user-pool?

### DISCUSSION THREAD

**Discussion by stepkirk** [cloud-custodian/cloud-custodian#7616] [AWS]
We normally enforce tags on AWS resources by using Custodian to look for certain required tags on a resource and then, if the tags don't exist or aren't in the correct format, we mark the resource for deletion after a certain grace period. With the Cognito identity-pool and user-pool resources, it doesn't look like we can check for tags the normal way and it doesn't look like we can mark a resource for later deletion. Is that true or am I missing something?

Any plans to add tagging/marking support in the future for these Cognito resources?

**Comment by thisisshi** [comment]
@stepkirk thanks for the question, looks like we need to augment our resource query to add the resource tags to the resource that will require an additional enhancement which will come out in a future release. the subsequent policy will look identical to other custodian tag policies, e.g.


```yaml
policies:
  - name: cognito-tags
    resource: aws.identity-pool
    filters:
      - tag:foo: bar
 ```

**Reply by kapilt** [reply]
We should convert to an issue for tracking
